solutions/src/main/java/ru/sberbank/school/task02/service/ExtendedFxConversionServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package ru.sberbank.school.task02.service;
+
+import ru.sberbank.school.task02.ExtendedFxConversionService;
+import ru.sberbank.school.task02.ExternalQuotesService;
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+public class ExtendedFxConversionServiceImpl extends FxConversionServiceImpl implements ExtendedFxConversionService {
+
+
+    ExtendedFxConversionServiceImpl(ExternalQuotesService externalQuotesService) {
+        super(externalQuotesService);
+    }
+
+    @Override
+    public Optional<BigDecimal> convertReversed(ClientOperation operation, Symbol symbol, BigDecimal amount, Beneficiary beneficiary) {
```

Comment:
Line is longer than 120 characters (found 135).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1942472525>*